x-pack/legacy/plugins/apm/public/context/UrlParamsContext/helpers.ts

```diff
@@ -61,10 +61,12 @@ export function getPathParams(pathname: string = '') {
   const pageName = paths[0];
 
   // TODO: use react router's real match params instead of guessing the path order
+
   switch (pageName) {
     case 'services':
       const servicePageName = paths[2];
       const serviceName = paths[1];
+      const serviceNodeName = paths[4];
```

Comment:
I hope we can get rid of this method soon-ish. Feels so hacky.

Reply:
Yeah, we're stretching it now, agreed.